css/style.css

```diff
@@ -216,3 +217,192 @@ ul {
 }
 
 
+
+
+
+/* Block about */
+.about {
+    padding: 0 375px 30px 375px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  
+}
+
+.about__title {
+    font-family: Roboto Slab, sans-serif;
+    font-size: 24px;
+    font-weight: bold;
+    margin-bottom: 60px;
+    margin-top: 100px;
+}
+
+.about__text {
+    margin-bottom: 28px;
+    font-family: Rubik, sans-serif;
+    font-size: 16px;
+}
+
+.about__box {
+    display: flex;
+    align-items: start;
+    gap: 40px;
+    margin-bottom: 150px;
+}
+
+.about__block {
+    display: inline;
+    align-items: center; 
+    text-align: center;
+}
+
+/* about__image */
+.about__image {
+    margin-bottom: 10px;
+    flex-shrink: 0;
+}
+
+.about__image-React {
+    width: 58px;
+    height: 52px;
+}
+
+.about__image-Redux {
+    width: 58px;
+    height: 55.23px;
+}
+
+.about__image-JS {
+    width: 52.63px;
+    height: 58px;
+}
+
+.about__image-Git {
+    width: 58px;
+    height: 58px;
+}
+
+.about__image-Mongo {
+    width: 25.92px;
+    height: 58px;
+}
+
+.about__image-MySQL {
+    width: 58px;
+    height: 56.82px;
+}
+
+
+
+
+
+.about__name {
+    font-family: Rubik, sans-serif;
+    font-size: 14px;
+    line-height: 20px;
+    color: #342B2B;
+    display: flex;
+    justify-content: center; 
+    text-align: center;
+}
+
+.box {
```

Comment:
where do you use this class? Why naming is not according to BEM?

Reply:
+